zebra/interface.c

```diff
@@ -2040,7 +2040,7 @@ DEFUN (bandwidth_if,
 	bandwidth = strtol(argv[idx_number]->arg, NULL, 10);
 
 	/* bandwidth range is <1-100000> */
-	if (bandwidth < 1 || bandwidth > 100000) {
+	if (bandwidth < 1 || bandwidth > 1000000) {
```

Comment:
This check can't possibly fail and this whole block should simply be removed

Reply:
ok